### DISCUSSION TITLE
Preview Crashing

### DISCUSSION THREAD

**Discussion by sherryycxie** [CS342/.github#25] [Technical Questions] [answered]
### In what area do you have a technical challenge?

Xcode, Simulator & Previews

### Description

I cloned a branch from the main branch of the Allergy group project; the preview of my project keeps crashing despite having built it successfully multiple times.

### Reproduction

I just built the program and clicked on the Home page and tried to preview it when the whole xcode project crashed.

### Expected behavior

It should not crash and give me a clear preview.

### Code of Conduct

- [X] I agree to follow this course's Code of Conduct and Contributing Guidelines

**Comment by PSchmiedmayer** [comment]
Hi @sherryycxie,
I have tried to verify the behavior in the CS342-Allergy application that you are working on, and it seems like previews are working for me there: 
<img width="1624" alt="Screenshot 2023-02-07 at 9 34 11 AM" src="https://user-images.githubusercontent.com/28656495/217320992-880385fd-8e21-429f-94be-df41b2241ff1.png">

Can you share the exact view (and maybe a link to your GitHub feature branch that you are working on, if it is a new view) and share the error message that you get when clicking the diagnostics button at the top of the failing preview?

**Reply by PSchmiedmayer** [reply, marked as answer]
@sherryycxie Wanted to shortly follow up on your question: We have added some improvements in the Template Application that will fix your issues. You can see the content of the PR here: https://github.com/StanfordBDHG/CardinalKitTemplateApplication/pull/7.

I will create a PR for your project as well so you can also get the updated code in your project as well.
